### ISSUE TITLE
Offset doesn't align to grid

### ISSUE THREAD

**Issue by sangdrax8** [knassher/FVTT-Teleport#32]
I am having an issue with the offset that is captured, not being the exact teleport point when used.  The new button to display where it will teleport to is correct, but the player is not placed inside the grid, and actually offset from it.  Moving in any direction will snap back to the grid, but this causes players to pass through walls etc.

I assumed this was due to the grid actually having an offset in the scene, but I have removed the offset of the grid, and I am still seeing this same error.

**Comment by sangdrax8** [comment]
Welll

**Comment by sangdrax8** [comment]
![Offset_Grid](https://user-images.githubusercontent.com/6662727/93358075-0f1d5580-f80f-11ea-9ea9-0cd91d6efc0c.png)
Here is a picture.  I saved an offset (offset 0) then teleported to the other scene, and came back.  When I come back (with out moving) I pulled up the same note again and saved where I was placed as point 1.  I then displayed them.

Issues in the picture:

1. Teleport point 0 is displayed where I wanted to arrive, but the character is well to the left
2. Even teleport point 1 shows aligned to the grid, but the character is clearly offset from that point as well.
